src/main/java/org/folio/rest/impl/ItemStorageAPI.java

```diff
@@ -143,22 +145,15 @@ public void deleteItemStorageItemsByItemId(
 
   private CompletableFuture<Item> setEffectiveCallNumber(Map<String, String> okapiHeaders, Context vertxContext, Item item) {
     CompletableFuture<Item> completableFuture = null;
-    EffectiveCallNumberComponents components = new EffectiveCallNumberComponents();
-    if (item.getItemLevelCallNumber() != null && !item.getItemLevelCallNumber().isEmpty()) {
-      components.setCallNumber(item.getItemLevelCallNumber());
-      item.setEffectiveCallNumberComponents(components);
+    if (StringUtils.isNotBlank(item.getItemLevelCallNumber()) && StringUtils.isNotBlank(item.getItemLevelCallNumberSuffix())) {
```

Comment:
I think, it make sense to extract this logic to a separate method (e.g. `shouldRetrieveHoldingsRecord`)

Reply:
Especially since were going to be adding the prefix to this condition soon.